docs/src/app/components/pages/customization/colors.jsx

```diff
@@ -139,13 +139,13 @@ const ColorsPage = React.createClass({
       backgroundColor: bgColor,
       color: fgColor,
       listStyle: 'none',
-      padding: '15px'
+      padding: '15px',
     };
 
     return (
-      <li style={styles}>{blockTitle}{bgColorText}</li>
+      <li style={styles}>{blockTitle} {bgColorText}</li>
```

Comment:
space here because it was giving syntax highlight issues in Webstorm for whatever reason